Example/Bellerophon/AppDelegate.swift

```diff
@@ -14,29 +14,33 @@ import Bellerophon
 class AppDelegate: UIResponder, UIApplicationDelegate, BellerophonManagerDelegate {
 
     var window: UIWindow?
-    let killSwitchURL = "http://qa.cms.prolific.io/killswitch/status/vandelay/ios"
+    
+    let killSwitchURL = "https://api.myjson.com/bins/19bad7"
+    let forceUpdateURL = "https://api.myjson.com/bins/13f3h7"
+    let perfectApp = "https://api.myjson.com/bins/s3uzf"
```

Comment:
Are they hosted using your account? Can you provide some steps on how to update these URLs in case they are not working anymore in the README?

Reply:
Just updated the README. Great catch